unstable/surface-layers.xml

```diff
@@ -115,12 +137,40 @@
       <description summary="sets a margin from the anchor point">
         Requests that the surface be placed some distance away from the anchor
         point on the output, in pixels.
+
+        Margin is double-buffered, see wl_surface.commit.
       </description>
-      <arg name="horizontal" type="uint"/>
-      <arg name="vertical" type="uint"/>
+      <arg name="horizontal" type="int"/>
+      <arg name="vertical" type="int"/>
     </request>
 
-    <enum name="input_devices">
+    <event name="configure">
+      <description summary="suggest a surface change">
+        The configure event asks the client to resize its surface.
+
+        The size is a hint, in the sense that the client is free to ignore it if
+        it doesn't resize, pick a smaller size (to satisfy aspect ratio or
+        resize in steps of NxM pixels).
```

Comment:
What do we do if the client is anchored on two opposite sides of the screen but chooses to use a size smaller than that? We should define that behavior.